platform/debian/do_debian_package.sh

```diff
@@ -38,20 +39,65 @@ if command_exists "${COMMAND}"; then
     {
         echo "Section: graphics"
         echo "Priority: optional"
-        echo "Depends: libsdl2-2.0-0"
+        echo "Depends: libsdl2-2.0-0, libc6 (>= 2.2.3)"
         echo "Architecture: ${ARCH}"
         echo "Version: ${VERSION}"
         echo "Installed-Size: $(du -ks "${INSTALL_DIR}/debian/usr/" | cut -f 1)"
-
         echo "Package: KOReader"
         echo "Maintainer: KOReader team"
         echo "Homepage: https://koreader.rocks"
-        echo "Description: An ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats"
-        echo " KOReader is a document viewer application, originally created for Kindle e-ink readers."
-        echo " It currently runs on Kindle, Kobo, PocketBook, Ubuntu Touch, Android and Linux devices"
+        echo "Description: Ebook reader application supporting PDF, DjVu, EPUB, FB2 and many more formats"
+        echo " KOReader is a document viewer application, originally created for"
+        echo " Kindle e-ink readers. It currently runs on Kindle, Kobo, PocketBook,"
+        echo " Ubuntu Touch, Android and Linux devices"
```

Comment:
I think the current main one is:

> KOReader is a document viewer for E Ink devices. Supported fileformats include EPUB, PDF, DjVu, XPS, CBT, CBZ, FB2, PDB, TXT, HTML, RTF, CHM, DOC, MOBI and ZIP files. It’s available for Kindle, Kobo, PocketBook, Android and desktop Linux.

(And if it's not, I/someone needs to update the website. It's not mentioning Cervantes for one.)

In any case you'd better not mention Ubuntu Touch since it doesn't actually start there. ;-)